assets/app/views/directives/osc-key-values.html

```diff
@@ -1,95 +1,175 @@
 <div ng-controller="KeyValuesController" class="labels">
     <div class="form-inline labels-edit" ng-show="editable">
-      <ng-form class="edit-label" name="form" novalidate>
-        <div class='form-group' ng-class="{'has-error': form.key.$error.oscKeyValid}">
-          <input class="form-control"
-                 type="text"
-                 name="key"
-                 ng-attr-placeholder="{{keyTitle}}"
-                 ng-model="key"
-                 autocorrect="off"
-                 autocapitalize="off"
-                 spellcheck="false"
-                 osc-input-validator="key">
-        </div>
-        <div class="form-group" ng-class="{'has-error': form.value.$error.oscValueValid}">
-          <input class="form-control"
-                 type="text"
-                 name="value"
-                 ng-attr-placeholder="{{valueTitle}}"
-                 ng-model="value"
-                 autocorrect="off"
-                 autocapitalize="off"
-                 spellcheck="false"
-                 osc-input-validator="value">
+      <form class="edit-label" name="form" novalidate>
+
+        <div layout row>
+
+          <div
+            flex grow="5"
+            class='form-group'
+            ng-class="{'has-error': form.key.$error.oscKeyValid}"
+            style="margin-right: 10px;">
+            <input class="form-control"
+                   type="text"
+                   name="key"
+                   ng-attr-placeholder="{{keyTitle}}"
+                   ng-model="key"
+                   autocorrect="off"
+                   autocapitalize="off"
+                   spellcheck="false"
+                   osc-input-validator="key">
+          </div>
+
+          <div
+            flex grow="5"
+            class="form-group"
+            ng-class="{'has-error': form.value.$error.oscValueValid}"
+            style="margin-right: 10px;">
+            <input class="form-control"
+                   type="text"
+                   name="value"
+                   ng-attr-placeholder="{{valueTitle}}"
+                   ng-model="value"
+                   autocorrect="off"
+                   autocapitalize="off"
+                   spellcheck="false"
+                   osc-input-validator="value">
+          </div>
+          <div
+            flex layout
+            grow="1" shrink="1" basis="auto"
```

Comment:
Create from template page @ wide width:
![screen shot 2016-02-15 at 10 26 32 am](https://cloud.githubusercontent.com/assets/280512/13052760/a4fe45ce-d3ce-11e5-929e-8e4ad0850d02.png)

Its made to fill the parent, so if the parent container is full page width, there would be a gap.  I suppose we could max width it, but I'm not sure that would help a ton.  `<osc-key-values class="max-width-class">` or `<div row><osc-key-values flex><div class="sibling" flex></div>` are a couple options that will probably work fine.